rules/windows/credential_access_lsass_memdump_handle_access.toml

```diff
@@ -17,38 +17,41 @@ index = ["winlogbeat-*", "logs-system.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "LSASS Memory Dump Handle Access"
-note = """## Triage and analysis.
+note = """## Triage and analysis
 
-### Investigating
+### Investigating LSASS Memory Dump Handle Access
 
 Local Security Authority Server Service (LSASS) is a process in Microsoft Windows operating systems that is responsible
-for enforcing the security policy on the system. It verifies users logging on to a Windows computer or server, handles
+for enforcing security policy on the system. It verifies users logging on to a Windows computer or server, handles
 password changes, and creates access tokens.
 
-Adversaries may attempt to access credential material stored in the process memory of the LSASS. After a user logs on,
-the system generates and stores a variety of credential materials in LSASS process memory. This is meant to facilitate
-single sign-on (SSO) ensuring a user isn’t prompted each time resource access is requested. These credential materials
-can be harvested by an adversary using administrative user or SYSTEM privileges to conduct Lateral Movement using
-[Use Alternate Authentication Material](https://attack.mitre.org/techniques/T1550/).
+Adversaries may attempt to access credential material stored in LSASS process memory. After a user logs on,the system
+generates and stores a variety of credential materials in LSASS process memory. This is meant to facilitate single
+sign-on (SSO) ensuring a user isn’t prompted each time resource access is requested. These credential materials can be
+harvested by an adversary using administrative user or SYSTEM privileges to conduct Lateral Movement using 
+[Alternate Authentication Material](https://attack.mitre.org/techniques/T1550/).
 
-#### Possible investigation steps:
+#### Possible investigation steps
 
-- Validate the correct install path for the process that triggered this detection
-- Confirm that any AV or EDR solutions that trigger this detection have the correct install path
+- Investigate the process execution chain (parent process tree).
+- Investigate other alerts associated with the user/host during the past 48 hours.
+- Validate the correct install path for the process that triggered this detection.
 
-### False Positive Analysis
+### False positive analysis
 
-- There should be very few if any false positives for this rule. However, it may be tripped by AV or EDR solutions.
+- There should be very few if any false positives for this rule. However, it may be tripped by AV or EDR solutions,
```

Comment:
```suggestion
- There should be very few if any false positives for this rule. However, it may be tripped by antivirus or endpoint detection and response solutions;
```